### DISCUSSION TITLE
DNS 转发的作用是什么，搭配passwall并开启DNS 转发，就不能访问国外网站了。

### DISCUSSION THREAD

**Discussion by fmsonic** [IrineSistiana/mosdns#761] [Q&A]
很奇怪，在mosdns里面关闭“DNS 转发”或者在opwrt“DHCP/DNS”手工设置dns为“127.0.0.1#5335”就一切正常。
passwall是远程dns设置为“通过UDP请求DNS”、“127.0.0.1:5335”

**Comment by cherinyy** [comment]
你应该在相对应的 OpenWrt LuCI 软件仓库提问，而不是在这里。

比如如果你使用的是 [sbwml/luci-app-mosdns](https://github.com/sbwml/luci-app-mosdns)，那应该在这个仓库中[提出议题](https://github.com/sbwml/luci-app-mosdns/issues/new/choose)。

从[代码](https://github.com/sbwml/luci-app-mosdns/blob/v5/luci-app-mosdns/root/etc/init.d/mosdns#L157-L174)来看，它的作用是设置 DNS 转发，取消重绑定保护（个人不推荐），忽略解析文件以及禁用 DNS 查询缓存。
